storey/redis_driver.py

```diff
@@ -0,0 +1,482 @@
+import json
+import math
+import os
+import time
+from datetime import datetime, timedelta, timezone
+from enum import Enum
+from typing import List, Union, Optional
+import pandas as pd
+import asyncio
+import concurrent
+from functools import partial
+from .drivers import Driver
+
+import redis
+import rediscluster
+from .dtypes import RedisError
+from .utils import schema_file_name
+
+
+class RedisType(Enum):
+    STANDALONE = 1
+    CLUSTER = 2
+
+
+class NeedsRedisAccess:
+    """Checks that params for access to Redis exist and are legal
+
+    :param webapi: URL to the web API (https or http). If not set, the REDIS_URL environment variable will be used.
+
+    """
+
+    def __init__(self, redis_url=None):
+        redis_url = redis_url or os.getenv('REDIS_URL')
+        if not redis_url:
+            self._redis_url = None
+            print('Missing redis_url parameter or REDIS_URL environment variable. Using fakeredit instead')
```

Comment:
Hm, we're using fakeredis in the implementation code? Isn't it only for tests?

Reply:
it is, fixed logic.